Dockerfile

```diff
@@ -79,6 +74,13 @@ RUN ./caldp-install-fitscut   /usr/local && \
    echo "/usr/local/lib" >> /etc/ld.so.conf && \
    ldconfig
 
+USER developer
```

Comment:
I'm a little worried about this as it used to be under root. What's the need for the change?

Reply:
pip was complaining about installing as root during the build.  AFAICT conda was installed as developer,  so I think we should honor the complaint and follow suit.

Reply:
Something this has raised for me though is why are we even upgrading pip at all? shouldn't we be leaving it alone?

Reply:
I propose deleting the whole block.  I don't see any reason to upgrade pip these days either.   Everything else is already a CALDP dependency and installed automatically.

I checked the base image and conda is installed as developer.   So I think our installs as root that pip is now complaining about were always (harmless) mistakes.   I propose doing all conda installs as developer.

I think we may be vulnerable to pip changing the baseline environment due to what we install.   caldp has dependencies and if they're not installed or are the wrong version I think the base will be changed.   The default pip upgrade policy is "only-if-needed" but I don't see anything more conservative like "never".   Maybe walking through with very explicit installs and --no-deps but we can only skip our deps if they don't matter in ops.

Reply:
I have had some similar concerns about the caldp install floating around at the back of my mind, and hoping there was never a reason for them to fly up to the front...

Reply:
I concur with deleting the whole block btw

Reply:
Reading through the Dockerfile,  it's scrambled in a few ways:
1.  yum is in two places
2. crds is in two places
3. certs are in two places
I think all of those can/should be unified into cohesive blocks to make it easier to understand.   I'll offer to do it now but maybe it should be ticketed so the risk is folded in with some other testing?   Or ignored.

Reply:
hmm. feels like a separate ticket. there is certainly a risk there of blocking these unrelated changes from getting merged.

Reply:
OK,  think I'm done here then.